plugin.video.infowars/default.py

```diff
@@ -46,14 +46,21 @@
 _artIcon		=_addon.get_icon(); 
 _artFanart	=_addon.get_fanart()
 __plugin__ = "Infowars"

```

Comment:
In your next update, please change the \__label\__  format to something else. That format should only be used for internal Python variables, not program variables